calibration/experiments/sphere_held_suarez_rhoe_equilmoist/observation_map.jl

```diff
@@ -0,0 +1,32 @@
+import EnsembleKalmanProcesses: TOMLInterface
+import ClimaCalibrate: observation_map, ExperimentConfig
+using ClimaAnalysis
+
+function observation_map(iteration)
+    ensemble_size = 10
+    output_dir = joinpath("output", "sphere_held_suarez_rhoe_equilmoist")
+    single_member_dims = 1
+    G_ensemble = Array{Float64}(undef, single_member_dims..., ensemble_size)
+    for m in 1:ensemble_size
+        member_path =
+            TOMLInterface.path_to_ensemble_member(output_dir, iteration, m)
+        simdir = SimDir(joinpath(member_path, "output_active"))
+        G_ensemble[:, m] .= process_member_data(simdir)
+    end
+    return G_ensemble
+end
+
+const meters = 1.0
+const days = 86400.0
+# Cut off first 120 day to get equilibrium, take second level slice
+function process_member_data(simdir)
+    isempty(simdir.vars) && return  NaN
+
+    ta = get(simdir; short_name = "ta", reduction = "average", period = "60d")
+    zonal_avg_temp_observations =
+        slice(average_lat(average_lon(ta)), z = 242meters)
+    sample_observation = Vector{Float64}(undef, 1)
+    sample_observation .=
+        slice(zonal_avg_temp_observations, time = 240days).data
```

Comment:
```suggestion
    sample_observation = slice(zonal_avg_temp_observations, time = 240days).data
```
Does this not work? Where is `slice` coming from? Can we qualify these methods?

Reply:
This works. `slice` and other methods come from ClimaAnalysis, I will qualify them.